src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuPaymentRequestServiceImpl.java

```diff
@@ -415,4 +417,39 @@ public String getPaymentRequestNoteTargetObjectId(String documentNumber) {
         return ((CuPaymentRequestDao) paymentRequestDao).getObjectIdByPaymentRequestDocumentNumber(documentNumber);
     }
 
+    /**
+     * This implementation uses the "DEFAULT_POS_APRVL_LMT" parameter to get the maximum value
+     * for which the PO amount can allow for automatic PREQ approval.
+     * 
+     * @see edu.cornell.kfs.module.purap.document.service.CuPaymentRequestService#isPurchaseOrderWithinAmountLimitForPaymentRequest(
+     * org.kuali.kfs.module.purap.document.PaymentRequestDocument)
+     */
+    @Override
+    public boolean isPurchaseOrderWithinAmountLimitForPaymentRequest(PaymentRequestDocument document) {

```

Comment:
Auto approval is referenced in a number of places within this method. Should the method name be "isPurchaseOrderWithinAmountLimitForAutoPaymentRequest" to reflect that?